scripts/ci/site/fix-markdown-docs.py

```
@@ -0,0 +1,291 @@
+            output.append("")
+
+        output.append(line)
+
+        if (
+            is_fence
+            and index + 1 < len(lines)
+            and lines[index + 1].strip()
+            and not lines[index + 1].strip().startswith("```")
+        ):
+            # Opening fence: blank line after language line only when next line starts body.
+            if stripped != "```" and not stripped[3:].strip():
```

Ruff (0.15.14)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)